### PR TITLE
December dependency updates

### PR DESCRIPTION
**Pull request by speedy1812**
This PR updates the dependencies, as well as [updating our Formspree forms](https://help.formspree.io/hc/en-us/articles/360056076314).

### CODE REVIEW

**Comment by speedy1812** [comment]
Before merging, we need to update the contact form for the Steeles.

File: source/steele.html.haml [hunk 1]
```diff
@@ -14,8 +14,10 @@ title: The Steele Family
   .bio-text
     = partial "markdown/steele-bio"
   .bio-contact
-    = partial "partials/contact-modal", locals: { button_title: "Contact the Steeles",
-      email: data.site.steele_email }
+    = link_to "https://ofreport.com/contact/", target: "_blank",
+      class: "button" do
+      = content_tag :button do
+        Contact the Steeles
```
Comment (joshukraine) [inline source/steele.html.haml:20]: Nice. 🙂

File: source/assets/javascripts/_formspree.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  xhr.onreadystatechange = function() {
+    if (xhr.readyState !== XMLHttpRequest.DONE) return;
+    if (xhr.status === 200) {
+      success(xhr.response, xhr.responseType);
+    } else {
+      error(xhr.status, xhr.response, xhr.responseType);
+    }
+  };
+  xhr.send(data);
+}
```
Comment (joshukraine) [inline source/assets/javascripts/_formspree.js:44]: Nice work here! 🙌🏻

**Review by joshukraine — APPROVED** [review]
Looks good! Thanks for your work on this! 👍🏻

**Pull request merged**